.svgo.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Migrating away from yaml to a modern configuration file:

https://github.com/svg/svgo#configuration
https://github.com/svg/svgo/issues/1504#issuecomment-886088967
https://github.com/svg/svgo/releases/tag/v2.0.0

Reply:
Was this config update tested? The changes appear to have broken the build between

before: https://61a6911fa4d94b15cad0d44a--zendeskgarden-react-components.netlify.app/?path=/story/components-avatars--default

after: https://61bab387d9e13be77d0a27ca--zendeskgarden-react-components.netlify.app/?path=/story/components-avatars--default

Note the absence of `focusable="false"` attribute on SVGs.

Reply:
Appears to be limited to Storybook and addressed under #1265 ([link](https://github.com/zendeskgarden/react-components/pull/1265/files#diff-cafe2123a72c4ce3a9f7e9ee4b0e188256eb02d7ec1e54fa2cadfac99b22f92b))